src/Commands/ProjectCreateCommand.php

```diff
@@ -139,6 +140,66 @@ public function validateCreateProject(CommandData $commandData)
         $this->providerManager()->validateCredentials();
     }
 
+    /**
+     * Get CI short name based on the CI provider.
+     */
+    public function getCiShortName($ci_provider) {
+        $reflect = new \ReflectionClass($ci_provider);
```

Comment:
Instead of reflection, maybe we should add a 'shortName' method to the provider, and call that?

Reply:
I'll fix this by renaming the folders in the template repo. That's what makes most sense.